Codeinwp/neve
# chore: move incompatibility notice to the dashboard page

### Summary
Moves incompatibility notice to neve dashboard page.


### Will affect visual aspect of the product

NO

### Test instructions


- When only the theme is installed, the notice should behave as before
- When both products are installed and upgradeable, both notices should appear and be yellow
- When both products are installed and just one is upgradeable, the notice should have a different message and be yellow
- Beware that the transients for updates get refreshed when you access the Themes / Plugins pages
-  You can test the various cases by changing the versions inside the file headers (`neve-pro-addon.php`/`style.css`)


Closes codeinwp/neve-pro-addon#723.
